scripts/commentOnPrAfterPreRelease.js

```diff
@@ -0,0 +1,118 @@
+module.exports = async ({ github, context, core }) => {
+  const owner = context.repo.owner;
+  const repo = context.repo.repo;
+
+  const prs = await getPRs({ github, repo, owner, ref: context.ref });
+  if (prs.length === 0) {
+    core.info("No PRs found");
+    return;
+  }
+  await createOrUpdateComment({
+    github,
+    repo,
+    owner,
+    prs,
+    context,
+  });
+};
+
+async function generatePRComment({
+  context,
+  github,
+  owner,
+  repo,
+  existingComment,
+}) {
+  if (!process.env.SUMMARY_JSON_STRING) {
+    const workflowRun = await github.rest.actions.getWorkflowRun({
+      owner,
+      repo,
+      run_id: context.runId,
+    });
+    const workflowRunUrl = workflowRun.data.html_url;
+    const quotedPreviousComment = quotePreviousComment(existingComment?.body);
+
+    const previousBuildStatus = quotedPreviousComment
+      ? `\nPrevious build information:\n${quotedPreviousComment}`
+      : "";
+    return `Could not publish Pre-release for ${process.env.COMMIT_SHA}. [View Logs](${workflowRunUrl})\nThe problem is likely in the \`NPM Publish\` or \`NPM CI\` step in the \`Trigger Pre-release Build\` Job.\n${previousBuildStatus}.\n `;
+  }
+  const summaryFileJson = JSON.parse(process.env.SUMMARY_JSON_STRING);
+
+  const releaseString = summaryFileJson
+    .map(releaseSummary => {
+      const { packageName, version } = releaseSummary;
+      return `  - ${packageName}@${version}\n`;
+    })
+    .join("");
+  const toInstallString = summaryFileJson
+    .map(releaseSummary => {
+      const { packageName, version } = releaseSummary;
+      return `${packageName}@${version}`;
+    })
+    .join(" ");
+  return `Published Pre-release for ${process.env.COMMIT_SHA} with versions:\n\`\`\`\n${releaseString}\`\`\`\n\nTo install the new version(s) run:\n\`\`\`\nnpm install ${toInstallString}\n\`\`\``;
+}
+
+async function getPRs({ github, repo, owner, ref }) {
+  const response = await github.rest.pulls.list({
+    repo,
+    owner,
+    head: `${owner}:${ref}`,
+  });
+  return response.data.map(pr => pr.number);
+}
+
+async function createOrUpdateComment({ github, repo, owner, prs, context }) {
```

Comment:
In the future we should use `Jobbie` to handle this comment but this can be done in the future